packaging/build.gradle

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'java'
+    id 'application'
+}
+
+configurations {
+    dbArtifacts
+    cliArtifacts
+    cliScripts
+}
+
+dependencies {
+    dbArtifacts(project(':ignite-runner'))
+    cliArtifacts(project(':ignite-cli'))
+    cliScripts(project(path: ':ignite-cli', configuration: 'cliScripts'))
+}
+
+task cliStartScript(type: CreateStartScripts) {
+    mainClassName = "org.apache.ignite.internal.cli.Main"
+    classpath = files(new File("../lib/${project(':ignite-cli').name}-${project(':ignite-cli').version}.jar"), new File("../lib/*"))
+    outputDir = startScripts.outputDir
+    applicationName = 'ignite3-cli'
+}
+
+distributions {
+    db {
+        startScripts.enabled = false
+        distributionBaseName = 'ignite3-db'
+        contents {
+            // create empty dirs
+            into('') {
+                File.createTempDir().with {
+                    ["log", "etc", "work"].each {new File(absolutePath, it).mkdirs() }
```

Comment:
Because Ignite 3 requires them to be passed as startup parameters. I can comment on this to make it clear.

Reply:
But it's not a startup script, it's a build script. I just don't understand how paths on the build server are relevant to the end user

Reply:
Because the startup script should not create any directory, they (required dirs) should be created by packaging mechanism. For example: 
- zip creates it inside
- rpm/deb preinstallation steps create these dirrectories
- docker image creates them 
- etc 

The startup script just passes them to the ignite and does its job – start ignite process, not create directories.

Reply:
> dpkg install (or yum) creates this directories as well   
yes

Reply:
Ok, I think it's ok for beta, but I don't know if pre-created directories are ok, because clients tend to use their own directories

Reply:
> Ok, I think it's ok for beta, but I don't know if pre-created directories are ok, because clients tend to use their own directories

They are pre-created only in zip and can be configured in `bootstrap-config` file. So, if a customer wants to change it – it can be done easily.